packages/ods/src/components/tooltip/src/components/ods-tooltip/ods-tooltip.tsx

```diff
@@ -54,19 +59,39 @@ export class OdsTooltip {
     this.triggerElement?.addEventListener('blur', this.boundHide);
     this.triggerElement?.addEventListener('focus', this.boundShow);
     this.triggerElement?.addEventListener('mouseenter', this.boundShow);
-    this.triggerElement?.addEventListener('mouseleave', this.boundHide);
+    this.triggerElement?.addEventListener('mouseleave', () => setTimeout(this.boundHide, 50));
```

Comment:
would probably be best to handle this on the hide code instead, cause this is working when going from the trigger to the tooltip but not the other way around.
To test this, go to the demo and move up and down multiple times, the tooltip is flickering when leaving it
<img width="138" alt="Capture d’écran 2024-12-30 à 09 21 06" src="https://github.com/user-attachments/assets/f7880b2b-b585-410d-b539-e53b87db559a" />